Packs/SecureWorks/Integrations/TaegisXDRv2/TaegisXDRv2.py

```diff
@@ -1531,7 +1570,10 @@ def update_investigation_command(client: Client, env: str, args=None):
                 f"The provided type, {args['type']}, is not valid for updating an investigation. "
                 f"Supported Type Values: {INVESTIGATION_TYPES}")
 
-        variables["input"][field] = args.get(field)
+        if field == "tags":
+            variables["input"]["tags"] = split_and_trim(args["tags"])
```

Comment:
split_and_trim is very similar to CommonServerPython argToList you can check it out if you like.

Reply:
I've gone ahead and replaced all uses of split_ant_trim with argToList